<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GRUP_KERNEL</class>
 <widget class="QDialog" name="GRUP_KERNEL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="100">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="90,1">
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QTreeWidget" name="tree_widget">
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_button_grubu_sil">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Grubu Sil</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/kernel_icons/kernel_sil_ikonu.png</normaloff>:/kernel_icons/kernel_sil_ikonu.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QToolButton" name="toolButton_help_button">
          <property name="text">
           <string>Yardim</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/kernel_icons/kernel_yardim_ikonu.png</normaloff>:/kernel_icons/kernel_yardim_ikonu.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="push_button_add_alt_grup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Alt Grup Ekle</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/kernel_icons/kernel_alt_grup_ekleme_ikonu.png</normaloff>:/kernel_icons/kernel_alt_grup_ekleme_ikonu.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="push_button_add_ana_grup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ana Grup Ekle</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/kernel_icons/kernel_ana_grup_ekleme_ikonu.png</normaloff>:/kernel_icons/kernel_ana_grup_ekleme_ikonu.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../KERNEL/kernel_icons.qrc"/>
 </resources>
 <connections/>
</ui>
